packages/docs/src/page-configs/ui-elements/accordion/examples/Menu.vue

```diff
@@ -0,0 +1,75 @@
+<template>
```

Comment:
I think with this example, we might mislead users with the `VaSidebar` component. Because we are duplicating `VaSidebar` functionality.

Reply:
Current docs `VaSidebar` also uses `VaAccordion`.